include/library.h

```diff
@@ -71,6 +71,7 @@ class Filter {
     std::string _query;
     bool _queryIsPartial;
     std::string _name;
+    std::string _flavour;
```

Comment:
This comment applies to the entire commit (titled "Allow to filter books by flavour.")

Please add a unit test `LibraryTest.filterByFlavour` in `test/library.cpp`.